pages/utilities/number-base-changer.tsx

```diff
@@ -11,6 +11,7 @@ import CallToActionGrid from "@/components/CallToActionGrid";
 import Meta from "@/components/Meta";
 import { Combobox } from "@/components/ds/ComboboxComponent";
 import NumberBaseChangerSEO from "@/components/seo/NumberBaseChangerSEO";
+import { ArrowLeftRight } from "lucide-react";
 
 export default function Base64Encoder() {
   const [fromBase, setFromBase] = useState(10);
```

Comment:
Instead of having two `useState` variables, let's store values as object.
It will make our lives easier when we need to switch values later.

```
const [base, setBase] = useState({ from: 10, to: 2 });
```